src/a_star_search/core.clj

```diff
@@ -1,24 +1,33 @@
 (ns a-star-search.core
   (:require [search.algorithms :as alg]
-            [problems.n-puzzle :as np])
+            [problems.n-puzzle :as np]
+            [problems.maze :as mz]
+            [clojure.tools.namespace.repl :refer [refresh]])
   (:gen-class))
 
+(def x "x") ;Pay no attention to the x behind the curtain. This varibale lets us use x without quotes.
```

Comment:
A nice little trick. It's unfortunate that you end up defining this twice. You could have done some `ns` magic to bring that `x` function over from your `problems.maze` namespace without the need to put a namespace in front of it so you'd only have to define it once, but it's not a major deal.